### PR TITLE
Fix: #14500 Added alt text to images for Userguide: credential_plugins.rst

### PR DESCRIPTION
**Pull request by maskboyAvi**
##### SUMMARY

Added alt text to images for Userguide: credential_plugins.rst

#14500 
##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

Latest

##### ADDITIONAL INFORMATION

Please add Hacktoberfest label to the PR. Thank You.

### CODE REVIEW

**Review by gundalow — CHANGES_REQUESTED** [review]
/home/runner/work/awx/awx/docs/docsite/rst/userguide/credential_plugins.rst:51: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/runner/work/awx/awx/docs/docsite/rst/userguide/credential_plugins.rst:60: WARNING: Explicit markup ends without a blank line; unexpected unindent.

**Review by oraNod — APPROVED** [review]
LGTM (apart from the broken docs build). Thanks @maskboyAvi

**Comment by maskboyAvi** [comment]
> /home/runner/work/awx/awx/docs/docsite/rst/userguide/credential_plugins.rst:51: WARNING: Explicit markup ends without a blank line; unexpected unindent. /home/runner/work/awx/awx/docs/docsite/rst/userguide/credential_plugins.rst:60: WARNING: Explicit markup ends without a blank line; unexpected unindent.

I will Fix the code ASAP.

**Comment by maskboyAvi** [comment]
@gundalow I have made the requested changes. Please check once ;)

**Review by gundalow — APPROVED** [review]
Thank you so much for doing these :)

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 1]
```diff
@@ -45,17 +45,20 @@ Use the AWX User Interface to configure and use each of the supported 3-party se
 3. For any of the fields below the **Type Details** area that you want to link to the external credential, click the |key| button of the input field. You are prompted to set the input source to use to retrieve your secret information.
 
 .. |key| image:: ../common/images/key-mgmt-button.png
-
+   :alt: Applications - Button for managing key-related operations.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Icon for managing external credentials
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 2]
```diff
@@ -45,17 +45,20 @@ Use the AWX User Interface to configure and use each of the supported 3-party se
...
 .. image:: ../common/images/credentials-link-credential-prompt.png 
+   :alt: Prompt for linking a credential field to Centrify Vault.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Credential section of the external secret management system dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 3]
```diff
@@ -45,17 +45,20 @@ Use the AWX User Interface to configure and use each of the supported 3-party se
...
 
 4. Select the credential you want to link to, and click **Next**. This takes you to the **Metadata** tab of the input source. This example shows the Metadata prompt for HashiVault Secret Lookup. Metadata is specific to the input source you select. See the :ref:`ug_metadata_creds_inputs` table for details.
 
 .. image:: ../common/images/credentials-link-metadata-prompt.png
+   :alt: Prompt for credential metadata
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Metadata section of the external secret management system dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 4]
```diff
@@ -45,17 +45,20 @@ Use the AWX User Interface to configure and use each of the supported 3-party se
...
 4. Select the credential you want to link to, and click **Next**. This takes you to the **Metadata** tab of the input source. This example shows the Metadata prompt for HashiVault Secret Lookup. Metadata is specific to the input source you select. See the :ref:`ug_metadata_creds_inputs` table for details.
 
 .. image:: ../common/images/credentials-link-metadata-prompt.png
+   :alt: Prompt for credential metadata 
 
 5. Click **Test** to verify connection to the secret management system. If the lookup is unsuccessful, an error message like this one displays:
 
 .. image:: ../common/images/credentials-link-metadata-test-error.png
-
+   :alt: Credentials link metadata Test Error
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example exception dialog for credentials lookup
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 5]
```diff
@@ -200,7 +203,7 @@ You need the Centrify Vault web service running to store secrets in order for th
 Below shows an example of a configured CyberArk AIM credential.
 
 .. image:: ../common/images/credentials-create-centrify-vault-credential.png 
-
+   :alt: Example of a Centrify Vault Credential Provider Lookup credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new centrify vault credential lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 6]
```diff
@@ -222,7 +225,7 @@ You need the CyberArk Central Credential Provider web service running to store s
 Below shows an example of a configured CyberArk CCP credential.
 
 .. image:: ../common/images/credentials-create-cyberark-ccp-credential.png 
-
+   :alt: Example of a configured CyberArk credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new CyberArk vault credential lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 7]
```diff
@@ -245,7 +248,7 @@ When **CyberArk Conjur Secrets Manager Lookup** is selected for **Credential Typ
 Below shows an example of a configured CyberArk Conjur credential.
 
 .. image:: ../common/images/credentials-create-cyberark-conjur-credential.png
-
+   :alt: Example of CyberArk Conjur Secrets Manager Lookup credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new CyberArk Conjur Secret lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 8]
```diff
@@ -268,7 +271,7 @@ When **HashiCorp Vault Secret Lookup** is selected for **Credential Type**, prov
 For more detail about Approle and its fields, refer to the `Vault documentation for Approle Auth Method <https://www.vaultproject.io/docs/auth/approle>`_.  Below shows an example of a configured HashiCorp Vault Secret Lookup credential.
 
 .. image:: ../common/images/credentials-create-hashicorp-kv-credential.png 
-
+   :alt: Example of a configured HashiCorp Vault Secret Lookup credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new HashiCorp Vault Secret lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 9]
```diff
@@ -292,7 +295,7 @@ For more detail about Approle and its fields, refer to the `Vault documentation
 Below shows an example of a configured HashiCorp SSH Secrets Engine credential.
 
 .. image:: ../common/images/credentials-create-hashicorp-ssh-credential.png 
-
+   :alt: Example of a configured HashiCorp Vault Signed SSH credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new HashiCorp Vault Signed SSH credential lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 10]
```diff
@@ -314,7 +317,7 @@ When **Microsoft Azure Key Vault** is selected for **Credential Type**, provide
 Below shows an example of a configured Microsoft Azure KMS credential.
 
 .. image:: ../common/images/credentials-create-azure-kms-credential.png
-
+   :alt: Example of a configured Microsoft Azure Key Vault credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new Microsoft Azure Key Vault credential lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 11]
```diff
@@ -334,6 +337,7 @@ When **Thycotic DevOps Secrets Vault** is selected for **Credential Type**, prov
 Below shows an example of a configured Thycotic DevOps Secrets Vault credential.
 
 .. image:: ../common/images/credentials-create-thycotic-devops-credential.png
+   :alt: Example of a configured Thycotic DevOps Secrets Vault credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new Thycotic DevOps Secrets Vault credential lookup dialog
```

File: docs/docsite/rst/userguide/credential_plugins.rst [hunk 12]
```diff
@@ -354,5 +358,6 @@ When **Thycotic Secrets Server** is selected for **Credential Type**, provide th
 Below shows an example of a configured Thycotic Secret Server credential.
 
 .. image:: ../common/images/credentials-create-thycotic-server-credential.png
+   :alt: Example of a configured Thycotic Secret Server credential.
```
Comment (oraNod) [inline docs/docsite/rst/userguide/credential_plugins.rst]:
```suggestion
   :alt: Example new Thycotic Secret Server credential lookup dialog
```

**Review by oraNod — APPROVED** [review]

**Review by tvo318 — APPROVED** [review]
Looks great, thanks @maskboyAvi 👍

**Pull request merged**